### PR TITLE
[Netatmo] Preserving outside temperature minimal ranges for dynamic channels

### PR DESCRIPTION
**Pull request by clinique**
Bringing a solution to issue #14256.
This solution is not the best but takes in account observations made in issue #13015.

Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/MeasureCapability.java [hunk 1]
```diff
@@ -69,26 +69,24 @@ private void updateMeasures(WeatherApi api, String deviceId, @Nullable String mo
...
+                    }
+
+                    MeasureConfiguration measureDef = channel.getConfiguration().as(MeasureConfiguration.class);
+                    String descriptor = channelTypeUID.getId().split("-")[0];
+                    try {
+                        Object result = measureDef.limit.isBlank()
+                                ? api.getMeasures(deviceId, moduleId, measureDef.period, descriptor)
+                                : api.getMeasures(deviceId, moduleId, measureDef.period, descriptor, measureDef.limit);
+                        MeasureClass.AS_SET.stream().filter(mc -> mc.apiDescriptor.equals(descriptor))
+                                .reduce((first, second) -> second)
```
Comment (clinique) [inline bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/MeasureCapability.java:83]: This is the important line

**Review by lolodomo — APPROVED** [review]
LGTM even if it looks more like a workaround than a clean solution.

**Comment by clinique** [comment]
> LGTM even if it looks more like a workaround than a clean solution.

You're correct, this is a workaround.

**Comment by lolodomo** [comment]
Temperature is negative now and I can confirm the fix is working.

**Pull request merged**